src/helpers/PageLoader.js

```diff
@@ -1,38 +1,38 @@
 import React from 'react';
 import config from '../magnolia.config';
-import { getAPIBase, getLanguages, removeCurrentLanguage, getCurrentLanguage, getVersion } from './AppHelpers';
+import { getAPIBase, getLanguages } from './AppHelpers';
 
 import { EditablePage } from '@magnolia/react-editor';
 import { EditorContextHelper } from '@magnolia/react-editor';
 
 class PageLoader extends React.Component {
   state = {};
 
-  getPagePath = () => {
-    const languages = getLanguages();
-    const nodeName = process.env.REACT_APP_MGNL_APP_BASE;
-    const currentLanguage = getCurrentLanguage();
-    let path = nodeName + window.location.pathname.replace(new RegExp('(.*' + nodeName + '|.html)', 'g'), '');
-
-    if (currentLanguage !== languages[0]) {
-      path = removeCurrentLanguage(path, currentLanguage);
-      path += '?lang=' + currentLanguage;
-    }
-
-    return path;
-  };
-
   loadPage = async (force) => {
     // Bail out if already loaded content.
     if (!force && this.state.pathname === window.location.pathname) return;
 
     const apiBase = getAPIBase();
+    const languages = getLanguages();
+    const spaRootNodePath = process.env.REACT_APP_MGNL_APP_BASE;
+    const pathName = window.location.pathname.replace(new RegExp('(.*' + spaRootNodePath + '|.html)', 'g'), '');
+    let relativePath = pathName;
+    const subIdTokenParam=`subid_token=${process.env.REACT_APP_MGNL_SUB_ID}`;
```

Comment:
Can we add the token param to the context after calling `EditorContextHelper.getMagnoliaContext()`?

Reply:
Yes. We can.